index.html

```diff
@@ -131,34 +134,34 @@ <h1 class="title">
   <td><a href="https://github.com/webxdc/webxdc_docs">webxdc/webxdc_docs</a></td>
 </tr>
 <tr>
-  <td colspan="3"><a href="https://github.com/orgs/webxdc/repositories?type=all">existing apps</a></td>
-</tr>
-<tr>
-  <td>hello world</td>
-  <td></td>
-  <td><a href="https://github.com/webxdc/hello">webxdc/hello</a></td>
+  <td>examples</td>
+  <td><a href="https://github.com/webxdc/hello">hello world</a></td>
+  <td><a href="https://github.com/orgs/webxdc/repositories?type=all">all</a></td>
 </tr>
 <tr>
-  <td colspan="3"><a href="https://www.youtube-nocookie.com/embed/I1K4pBvb2pI">screencast — just web apps</a></td>
+  <td>screencast</td>
+  <td colspan="2"><a href="https://www.youtube-nocookie.com/embed/I1K4pBvb2pI">&quot;just web apps&quot;</a></td>
 </tr>
 
 <!-- BOTS -->
 
 <tr>
-  <th colspan="3"><a href="https://github.com/deltachat-bot">bots</a></th>
+  <th colspan="3"><a href="https://github.com/deltachat-bot">Bots</a></th>
 </tr>
 <tr>
   <td>website</td>
   <td><a href="https://bots.delta.chat">bots.delta.chat</a></td>
   <td><a href="https://github.com/deltachat-bot/bot-pages">deltachat-bot/bot-pages</a></td>
 </tr>
 <tr>
-  <td>echo bot</td>
-  <td></td>
-  <td><a href="https://github.com/deltachat-bot/echo">deltachat-bot/echo</a></td>
+  <td>examples</td>
+  <td><a href="https://github.com/deltachat-bot/echo">echo</a></td>
+  <td><a href="https://support.delta.chat/c/bots/">all</a></td>
```

Comment:
```suggestion
  <td><a href="https://support.delta.chat/c/bots/">more bots</a></td>
```

I try to use unique link text for accessibility reasons.